.github/workflows/stargz_tests.yml

```diff
@@ -0,0 +1,79 @@
+name: vHive stock-only stargz tests
+
+on:
+  push:
+    branches: [ main ]
+    paths-ignore:
+    - 'docs/**'
+    - '**.md'
+    - 'function-images/**'
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+    - 'docs/**'
+    - '**.md'
+    - 'function-images/**'
+  workflow_dispatch:
+
+env:
+  GOOS: linux
+  GO111MODULE: on
+
+jobs:
+  stargz-container-test:
+    name: Test running stargz-based image using kn
+    env:
+      KIND_VERSION: v0.14.0
+      K8S_VERSION: v1.23
+      YAML_DIR: workloads/container
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        service:
+          [
+            trace_func_go,
+          ]
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          lfs: "true"
+      - name: Checkout LFS objects
+        run: git lfs checkout
+
+      - name: Set up Go 1.19
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+
+      - name: Create k8s Kind Cluster
+        run: bash ./scripts/stargz/01-kind.sh
+
+      - name: Install Serving
+        run: bash ./scripts/stargz/02-serving.sh
+
+      - name: Install Kourier
+        run: bash ./scripts/stargz/02-kourier.sh
```

Comment:
so should we remove the 2-contour file?

Reply:
Yes, it was leftover after the changes done to the workflow. Removed, thanks for specifying it.